src/helpers/getByAPI.js

```diff
@@ -9,9 +9,32 @@ import {
   throwRenamedFunctionError,
 } from './errors';
 
+export type TextMatchOptions = {
+  exact: boolean,
+};
+
+const DEFAULT_TEXT_MATCH_OPTIONS: TextMatchOptions = {
+  exact: true,
+};
+
 const filterNodeByType = (node, type) => node.type === type;
 
-const getNodeByText = (node, text) => {
+const matchText = (
```

Comment:
Let's use the original implementation, which can be found here: https://github.com/testing-library/native-testing-library/blob/master/src/lib/matches.js. This will also solve the missing normalization, which I believe we can implement, even just to be more compatible with testing library in general, it's not too much code really

Reply:
OK, I'll go with that implementation but in slightly modified manner: we don't have a matcher defined as predicate function option implemented